how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md

```diff
@@ -7,6 +7,12 @@ description: >-
 
 # Upgrade to v8
 
+{% hint style="warning" %}
+
+Please be aware that while we make every effort to ensure that our platform updates are broadly compatible and offer detailed instructions for upgrading, we cannot guarantee that custom code developed by our customers will always be compatible with new versions of our software. This includes any custom modifications or extensions to core functionalities, such as method overrides or custom integrations. It is the responsibility of our customers to review and test their custom code to ensure compatibility with each new version. Our team provides comprehensive upgrade guides to assist in this process, but these cannot encompass the unique customizations that may be present in each customer's environment. Therefore, we strongly recommend establishing a thorough testing protocol for your specific customizations to safeguard against potential issues during the upgrade process.
+
+{% endhint %}
```

Comment:
A few comments:
- I think we should have this in any upgrade note (not only v8), at least for Ruby on Rails.
- "we" -> "Forest Admin"
- "our customers" -> "integrators", "developers" or similar? because not necessarily customers.

It is a lot to read to begin your upgrade but it as the benefit to be complete and clear.

Reply:
> I think we should have this in any upgrade note (not only v8), at least for Ruby on Rails.

This is the plan, but instead of copy pasting something that'll most likely changes, I prefer to put this here once and then copy/paste when we all agrees.

The 2 others comments should be fixed.

Reply:
sorry I missed your PR comment 🙏